source/specs.txt

```diff
@@ -5,179 +5,180 @@ MongoDB Drivers Specs
-     - |checkmark|
-     -
-     -
-   * - :spec:`Transactions </transactions/transactions.rst>`
-     - |checkmark|
-     -
-     -
-   * - :spec:`Polling SRV Records for mongos Discovery </polling-srv-records-for-mongos-discovery/polling-srv-records-for-mongos-discovery.rst>`
-     - |checkmark|
-     -
-     -
-   * - :spec:`URI options </uri-options/uri-options.rst>`
-     - |checkmark|
-     -
-     -
-   * - :spec:`Convenient API for Transactions  </transactions-convenient-api/transactions-convenient-api.rst>`
-     - |checkmark|
-     -
-     -
-   * - :spec:`Connection Monitoring and Pooling </connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst>`
-     - |checkmark|
-     -
-     -
-   * - :spec:`Connection String </connection-string/connection-string-spec.rst>`
-     -
-     - |checkmark|
-     -
-   * - :spec:`CRUD operations </crud/crud.rst>`
-     -
-     - |checkmark|
-     -
-   * - :spec:`Index Management </index-management.rst>`
-     -
-     - |checkmark|
-     -
-   * - :spec:`Find, getMore and killCursors commands </find_getmore_killcursors_commands.rst>`
-     -
-     - |checkmark|
-     -
-   * - :spec:`Driver Bulk update </driver-bulk-update.rst>`
-     -
-     - |checkmark|
-     -
-   * - :spec:`Enumerate Collections </enumerate-collections.rst>`
-     -
-     - |checkmark|
-     -
-   * - :spec:`Out aggregation pipeline operator </out-aggregation-pipeline-operator.rst>`
-     -
-     - |checkmark|
-     -
-   * - :spec:`Server write commands	</server_write_commands.rst>`
-     -
-     - |checkmark|
-     -
-   * - :spec:`Logical Sessions </sessions/driver-sessions.rst>`
-     -
-     - |checkmark|
-     -
-   * - :spec:`Retryable Writes </retryable-writes/retryable-writes.rst>`
-     -
-     - |checkmark|
-     -
-   * - :spec:`Change Streams </change-streams/change-streams.rst>`
-     -
-     - |checkmark|
-     -
-   * - :spec:`Causally Consistent Reads </causal-consistency/causal-consistency.rst>`
-     -
-     - |checkmark|
-     -
-   * - :spec:`Enumerate Databases </enumerate-databases.rst>`
-     -
-     - |checkmark|
-     -
-   * - `OP_QUERY, OP_GET_MORE, OP_KILL_CURSOR, OP_REPLY <http://docs.mongodb.org/meta-driver/latest/legacy/mongodb-wire-protocol/>`_
-     - |checkmark|
-     -
-     -
-   * - :manual:`OP_INSERT, OP_UPDATE, OP_REMOVE, getLastError	</reference/command/getLastError/>`
-     - |checkmark|
-     -
-     -
-   * - :spec:`Command Monitoring </command-monitoring/command-monitoring.rst>`
-     - |checkmark|
-     -
-     -
-   * - :spec:`SDAM Monitoring </server-discovery-and-monitoring/server-discovery-and-monitoring-monitoring.rst>`
-     - |checkmark|
-     -
-     -
-   * - :spec:`GridFS </gridfs/gridfs-spec.rst>`
-     -
-     - |checkmark|
-     -
-   * - :spec:`OP_COMPRESSED </compression/OP_COMPRESSED.rst>`
-     - |checkmark|
-     -
-     -
-
-.. include:: /includes/unicode-checkmark.rst
+..
```

Comment:
You could just remove this rather than commenting it out since it's in git history.

Reply:
Good point. I'm also going to include another change that somehow didn't get picked up but should have been